verdict/main.go

```diff
@@ -57,13 +60,18 @@ func handleCompiling(language string, source *multipart.FileHeader) ([]byte, err
 	if err != nil {
 		return nil, err
 	}
+
 	req.Header.Set("Content-Type", writer.FormDataContentType())
 	client := &http.Client{}
 	res, err := client.Do(req)
 	if err != nil {
 		return nil, err
 	}
 
+	if res.StatusCode != http.StatusOK {
+		return nil, compilationError
```

Comment:
Log situations like these, they'll probably help debug your application!

Even though this is a normal flow of the application, we don't expect _very_ high numbers of compilation errors.